samples/queryParamsNamedTypes.js

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(
+  projectId = 'my_project',
+  datasetId = 'my_dataset',
+  tableId = 'my_table'
+) {
+  // [START bigquery_query_params_named_types]
+  // Import the Google Cloud client library
+  const {BigQuery} = require('@google-cloud/bigquery');
+  const bigquery = new BigQuery();
+
+  async function queryParamsNamedTypes() {
+    // Run a query using named query parameters and provided parameter types.
+
+    /**
+     * TODO(developer): Uncomment the following lines before running the sample.
+     */
+    // projectId = process.env.PROJECT_ID;
```

Comment:
In the stye guide, we use a hard coded string for projectId:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head

Reply:
Okay, got it! But now I need to figure out where I saw this as an example and update it...